Index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Page Title</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.2.1/css/fontawesome.min.css" integrity="sha384-QYIZto+st3yW+o8+5OHfT6S482Zsvz2WfOzpFSXMF9zqeLcFV0/wlZpMtyFcZALm" crossorigin="anonymous">
+</head>
+<body> 
+    <div class="wrapper">
+      <section class="section1">
+       <header class="toolbar">           
+            <span id="logo">My Logo</span> 
+            <span id="switch">               
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>             
+            </span>                  
+       </header>
+       </section>
+       <section class="section2">
+        <div id="greetings">I’m Alexandra. Glad to see you!</div>
+        <div id="details">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</div>
+        <footer>
+            <span> LET’S CONNECT</span>
+            <div class="icon-bar">                
+                <a href="#"><img  src="./images/twitter.png"></a>
+                <a href="#"><img  src="./images/linkedin.svg">
+                <a href="#"><img  src="./images/m.png"></a>
+                <a href="#"><img  src="./images/git.png"></a> 
+                <a href="#"><img  src="./images/peace.png"></a>                  
+            </div>        
```

Comment:
Instead of using `div` again I would recommend using a `ul` tag since it is a list of icons. Then you can wrap each icon inside a `li` tag which would make it a proper list of icons. Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.